lib/twitter-ads/client.rb

```diff
@@ -3,7 +3,7 @@
 
 module TwitterAds
 
-  API_VERSION = '3'.freeze
+  API_VERSION = '4'.freeze
```

Comment:
Lets leave this as `3` for now, given that this change will break all deprecated endpoints. We'll make the change once all the other v4 changes have been merged.